content/en/security_platform/application_security/_index.md

```diff
@@ -1,17 +1,48 @@
 ---
-title: Monitoring Threats with Datadog Application Security
+title: Application Security
 kind: documentation
+description: Monitor threats targeting production system, leveraging the execution context provided by distributed traces.
+is_beta: true
+disable_sidebar: true
 further_reading:
 - link: "/security_platform/guide/how-appsec-works/"
   tag: "Guide"
-  text: "How Datadog Application Security Monitoring Works"
+  text: "Learn how Application Security works"
 - link: "/security_platform/application_security/setup_and_configure/#compatibility"
   tag: "Documentation"
-  text: "Programming Language and Framework Compatibility"
+  text: "Learn more about language and framework compatibility"
 ---
 
+<div class="alert alert-warning">
+Application Security is currently in public beta. See the <a href="https://app.datadoghq.com/security/appsec">in-app instructions</a> to get started.
+</div>
 
+{{< img src="/security_platform/application_security/app-sec-landing-page.png" alt="The Application Security landing page in Datadog, which displays your services, suspicious requests, and detected signals" width="75%">}}
 
-## Further Reading
+Datadog Application Security monitors application-level threats to identify attacks that target code-level vulnerabilities, such as Server-Side-Request-Forgery (SSRF), SQL injection, Log4Shell, and Reflected Cross-Site-Scripting (XSS).
 
-{{< partial name="whats-next/whats-next.html" >}}
\ No newline at end of file
+Application Security leverages Datadog [tracing libraries][1], the [Datadog Agent][2], and in-app detection rules to detect threats in your application environment and trigger signals whenever a vulnerability or attack occurs.
```

Comment:
An attack targets your production system, or a vulnerability is triggered from the code